core/arch/arm/plat-imx/conf.mk

```diff
@@ -30,122 +30,130 @@ mx7-flavorlist = \
 	mx7swarp7 \
 
 ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6ul-flavorlist)))
+$(call force,CFG_MX6,y)
 $(call force,CFG_MX6UL,y)
+$(call force,CFG_TEE_CORE_NB_CORE,1)
+include core/arch/arm/cpu/cortex-a7.mk
 else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6ull-flavorlist)))
+$(call force,CFG_MX6,y)
 $(call force,CFG_MX6ULL,y)
+$(call force,CFG_TEE_CORE_NB_CORE,1)
+include core/arch/arm/cpu/cortex-a7.mk
 else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6q-flavorlist)))
+$(call force,CFG_MX6,y)
 $(call force,CFG_MX6Q,y)
+$(call force,CFG_TEE_CORE_NB_CORE,4)
 else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6d-flavorlist)))
+$(call force,CFG_MX6,y)
 $(call force,CFG_MX6D,y)
+$(call force,CFG_TEE_CORE_NB_CORE,2)
 else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6dl-flavorlist)))
+$(call force,CFG_MX6,y)
 $(call force,CFG_MX6DL,y)
+$(call force,CFG_TEE_CORE_NB_CORE,2)
 else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6s-flavorlist)))
+$(call force,CFG_MX6,y)
 $(call force,CFG_MX6S,y)
+$(call force,CFG_TEE_CORE_NB_CORE,1)
 else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6sx-flavorlist)))
 $(call force,CFG_MX6,y)
 $(call force,CFG_MX6SX,y)
-$(call force,CFG_IMX_UART,y)
+$(call force,CFG_TEE_CORE_NB_CORE,1)
 else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx7-flavorlist)))
```

Comment:
can you split mx7 flavor into mx7d and mx7s flavor like it is done for i.mx6?
you could keep the CFG_IMX7 in both cases.

Reply:
There is really no much difference between i.mx7d and i.mx7s. currently the only difference is one is 2 cores, the core is 1 core. So no need to split.

Reply:
It is the same thing as the difference between 6q and 6d.
splitting 7 into 7s and 7d does not add a lot of code, and brings consistency and correctness to the conf.mk file

Reply:
Actually there is no strong reason to add new entry for 6Q/6D, the difference is just core number.
waste a few bytes does no harm. Same to i.MX7D/S.